.github/workflows/build.yaml

```diff
@@ -45,11 +45,15 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - run: |
+          rm MODULE.bazel
+          mv MODULE.bazel.windows MODULE.bazel
```

Comment:
note, I struggled for a couple hours to find any way to turn off the zig toolchain with flags. Ended up with this gross hack, at least it's simple?

Reply:
it could be because the zig toolchain is registered in the hermic toolchain `MODULE.bazel` which should be fixed here: https://github.com/uber/hermetic_cc_toolchain/commit/f8c17de30fbf185cc5c2f61b2b508aa3b3d294a1 but hasn't been released yet

Reply:
Yes! Thank you, exactly that.

Note that it *ought* to be fine for it to register toolchains, since the end-user in the root module can always choose overrides of those toolchains. However I didn't figure out how. Maybe they could have just documented that. Removing the root registration is fine too, if they're okay with a breaking change.